packages/rdbg/rdbg.1.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "erwan.jahier@imag.fr"
+bug-reports: "erwan.jahier@imag.fr"
+authors: [ "Erwan Jahier" ]
+license: "GPL-3"
+homepage: "http://rdbg.forge.imag.fr/"
+dev-repo: "https://forge.imag.fr/anonscm/git/rdbg/rdbg.git"
+build: [
+  ["oasis" "setup"]
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  ["ocamlfind" "remove" "rdbg-plugin"]
+]
+build-doc: [ "ocaml" "setup.ml" "-doc" ]
+depends: [
+  "base-unix"
+  ("camlp4" {= "4.03.0"} | "camlp4" {>= "4.02.1+system"} | "camlp4" {= "4.02.1+2"} | "camlp4" {= "4.02.1+1"} | "camlp4" {= "4.02.1"} | "camlp4" {= "4.02.0-1modular-implicits"} | "camlp4" {= "4.02.0+system"} | "camlp4" {= "4.02.0+2"} | "camlp4" {= "4.02.0+1"} | "camlp4" {= "4.02.0"} | "camlp4" {= "4.01.0"})
```

Comment:
Previously 6/8 builds passed due to uninstallability. In order to test packages that are only compatible with some of our test configurations, uninstallability of a package is not a failure. For instance, your package is not installable on OCaml 4.00.1 but it still "passes" the test on this platform by having a clean behavior (informing the user it isn't compatible).

I don't know what `GnuplotRif` is or why it is necessary but if the package is not expected to work on OS X, you can restrict it in the opam metadata so that OS X users have a better experience.

Your conception of the publishing process is correct. We won't typically merge packages which have build failures unless it is due to a deficiency of the test machines (e.g. a new enough system package isn't available). In all cases, it is faster and easier to everyone if the package submitter investigates the Travis build failures and corrects them without prompting.